environment/src/test/java/com/sequenceiq/environment/environment/poller/FreeIpaPollerProviderTest.java

```diff
@@ -136,7 +136,7 @@ private static Stream<Arguments> freeIpaSyncStatuses() {
     public static Stream<Arguments> upgradeCcmStatuses() {
         return Stream.of(
                 Arguments.of(OperationState.COMPLETED, AttemptState.FINISH, ""),
-                Arguments.of(OperationState.FAILED, AttemptState.BREAK, "FreeIpa Upgrade CCM failed."),
+                Arguments.of(OperationState.FAILED, AttemptState.BREAK, "FreeIpa Upgrade CCM failed: null"),
```

Comment:
hmmm. not too nice.

Reply:
forgot to include an actual error message, fixed it